config/locales/de.yml

```diff
@@ -142,6 +154,17 @@ de:
       away: "Team 2"
       score: "Ergebnis"
       points: "Punkte"
+    gamemode:
+      ko: "KO"
+      ko_group: "Gruppen-KO"
+      double_elimination: "Doppel-KO-System"
+      round_robin: "Jeder gegen Jeden"
+      two_halfs: "Hin- und Rückrunde"
+      swiss: "Schweizer System"
+      danish: "Dänisches System"
+      elo: "ELO"
+      win_loss: "Gewinne-Niederlagen"
+      true_skill: "Wahres Können"
```

Comment:
I would personally not translate true skill at all, since it's a specific way of calculating the ranking data and nobody knows what "Wahres Können" is representing.